test/onnx/verify/dynamicquantizelinear_2d_test.cpp

```diff
@@ -52,3 +53,32 @@ TEST_CASE(dynamicquantizelinear_2d_test)
     std::vector<uint8_t> y_zpt_gold = {0};
     EXPECT(migraphx::verify::verify_rms_range(y_zpt, y_zpt_gold));
 }
+
+TEST_CASE(dynamicquantizelinear_2d_simplify_test)
+{
+    auto p   = migraphx::parse_onnx("dynamicquantizelinear_2d_test.onnx");
+    auto* mm = p.get_main_module();
+    migraphx::run_passes(*mm, {migraphx::simplify_dynamicquantizelinear{}});
+    p.compile(migraphx::make_target("ref"));
+
+    std::vector<float> data{1.0, 2.1, 1.3, 2.5, 3.34, 4.0, 1.5, 2.6, 3.9, 4.0, 3.0, 2.345};
```

Comment:
Please add a couple of negative values as well.. unless there is some other logic here..

Reply:
Thats fine, no harm in adding the additional test. the intent with these was to copy the non simplified test with the same input data, add the simplify pass  and get a result that's just shifted by -128 after all the quantization is done. It confirms that the same input + a pass shifts the data correctly and as expected.

Reply:
> It confirms that the same input + a pass shifts the data correctly and as expected.
Thanks, that is why I had suggested if you could in a single test verify two golden sets: one before the modification, and one after.

Reply:
Yup all good. I added these as a side by side and also compared scale values as well to verify that if we parse in the same graph, perform a pass on one, the outputs should be shifted to the correct range, scales shouldn't change and zero point should be shifted to the proper value.